gradle/plugins/src/main/kotlin/com.hedera.gradle.versions.gradle.kts

```diff
@@ -24,3 +24,5 @@ plugins {
 group = "com.hedera.hashgraph"
 
 javaPlatform { allowDependencies() }
+
+tasks.register("releaseMavenCentral")
```

Comment:
Ahh, so this was needed after all, even though it is also in *.lifecycle.grade.kts. GTK.